verilog/tools/preprocessor/verilog_preprocessor_test.sh

```diff
@@ -560,5 +569,241 @@ diff --strip-trailing-cr -u "$MY_EXPECT_FILE" "$MY_OUTPUT_FILE" || {
   exit 1
 }
 
+################################################################################
+echo "=== Test multiple-compilation-unit: including a file with an absolute path"
+
+cat > "$MY_INPUT_FILE" <<EOF
+\`include "${MY_ABSOLUTE_INCLUDED_FILE_1}"
+input_content
+EOF
+
+cat > "$MY_ABSOLUTE_INCLUDED_FILE_1" <<EOF
+included1_content
+EOF
+
+
+cat > "$MY_EXPECT_FILE" <<EOF
+${MY_INPUT_FILE}:
+(#293: "included1_content")
+(#293: "input_content")
+
+EOF
+
+"$preprocessor" multiple-compilation-unit "$MY_INPUT_FILE" > "$MY_OUTPUT_FILE" 2>&1
+
+status="$?"
+
+[[ $status == 0 ]] || {
+  "Expected exit code 0, but got $status"
+  exit 0
+}
+
+diff --strip-trailing-cr -u "$MY_EXPECT_FILE" "$MY_OUTPUT_FILE" || {
+  exit 1
+}
+
+################################################################################
+echo "=== Test multiple-compilation-unit: including a file with a relative path with the correct incdir"
+
+cat > "$MY_INPUT_FILE" <<EOF
+\`include "${MY_RELATIVE_INCLUDED_FILE_1}"
+input_content
+EOF
+
+cat > "$MY_ABSOLUTE_INCLUDED_FILE_1" <<EOF
+included1_content
+EOF
+
+
+cat > "$MY_EXPECT_FILE" <<EOF
+${MY_INPUT_FILE}:
+(#293: "included1_content")
```

Comment:
I think at some point we should emit

```verilog
`line "filename"
```
tokens, for further tools down the line to deal with it properly (I am not sure that verible will deal with it in itself, so that should probably be recorded in some issue).

It essentially tells tools that read that output stream that "_the following content actually comes from this filename_

(This similar to `#line` from the C preprocessor) 
[So at this point, mostly request for issue creating]

Reply:
I opened this issue #1484 for `` `line`` support.
Do you mean Verible should manually add `` `line`` in files? or we just need to support it if found in source files.

Reply:
Nice thanks! I'll comment on that issue for details that come to mind.

(One unrelated question: how did you make the single back-tick in the `line` line ? I failed to get something like that with the github markdown...)

Reply:
It's `line included between 2 double back-ticks, the thing is there should be a space between the double back-tick and the back-tick of line.

Reply:
`` `test `` :) Ah, thank you!